### DISCUSSION TITLE
Error with name of the tables when I did npx prisma db pull

### DISCUSSION THREAD

**Discussion by martinGM05** [prisma/prisma#24341] [Q&A]
### Question

I have a problem when I do `npx prisma db pull`.

This create the schemas very well but when I want to use the client, the schema PING should show me like a 'PING' but just view pING.

```typescript 
model PING {
  PING_ID   BigInt    @id @default(autoincrement())
  PING_TIME DateTime? @default(now()) @db.DateTime(0)
}
```

```typescript 
import { PrismaClient } from '@prisma/client'
import { PingModel } from '../commons/interfaces/Ping'
import Logger from '../commons/utils/logger'

const prisma = new PrismaClient()
const logger = new Logger()

export class UserDAO {
  static async getUser(): Promise<PingModel[]> {
    await logger.info('DAO: Starting getUser')
    const ping = await prisma.pING.findMany()

    await logger.info('DAO: Ending getUser')
    return ping
  }
}

- OS: Windows
- Database: MySQL
- Node.js version:  v20.11.0

- Prisma v5.14.0

**Comment by ludralph** [comment]
Hi @martinGM05 

Thank you for raising this question.

Prisma ORM has a naming convention of Camel Casing. When you introspected, it generated `PING` model in the schema and when using it in the Client, you would need to access it like `prisma.pING`. If you would like to change this, the you can rename the model in the schema and use the `@@map` attributes to map it to the database name. See the [documentation](https://www.prisma.io/docs/orm/prisma-client/setup-and-configuration/custom-model-and-field-names#example-relational-database) for more information.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
